RealmBrowser/Classes/RLMRealmBrowserWindowController.m

```diff
@@ -101,6 +101,20 @@ - (IBAction)saveObjcModels:(id)sender
     [RLMModelExporter saveModelsForSchemas:objectSchemas inLanguage:RLMModelExporterLanguageObjectiveC];
 }
 
+- (IBAction)saveCopy:(id)sender
+{
+    NSString *fileName = [self.modelDocument.presentedRealm.realm.path lastPathComponent];
+    NSSavePanel *panel = [NSSavePanel savePanel];
+    [panel setNameFieldStringValue:fileName];
+    [panel beginSheetModalForWindow:self.window completionHandler:^(NSInteger result){
+        if (result == NSFileHandlingPanelOKButton)
+        {
+            NSURL *fileURL = [panel URL];
+            [self.modelDocument.presentedRealm.realm writeCopyToPath:fileURL.path error:nil];
```

Comment:
Is doing `-compact` implicitly a good idea? The user may just want to save a copy of their Realm with free space removed, not with the other changes that `-compact` makes to the structure.